includes/admin/tools/export/give-export-donations-exporter.php

```diff
@@ -374,6 +378,10 @@ public function get_data() {
 					$data[ $i ]['address_country'] = isset( $address['country'] ) ? $address['country'] : '';
 				}
 
+                if ( ! empty( $columns['phone'] ) ) {
+                    $data[ $i ]['phone'] = $payment_meta['_give_payment_donor_phone'];
+                }
+
```

Comment:
The unreleased tag about this change is missing on the top of this function.

Reply:
Resolved: db7a057b6c1314642c5aa2330356ae9721f2c47a